akka-docs-dev/rst/java/code/docs/http/javadsl/HandlerExampleDocTest.java

```diff
@@ -12,7 +12,7 @@
 import akka.http.javadsl.testkit.TestRoute;
 import org.junit.Test;
 
-public class HandlerExampleSpec extends JUnitRouteTest {
+public class HandlerExampleDocTest extends JUnitRouteTest {
```

Comment:
changed a few of those to Test. In general "java tests" = Test, "scala test" = Spec.